src/modules/authorization/reducer.spec.ts

```diff
@@ -81,3 +84,108 @@ describe.each([
     })
   }
 })
+
+describe('authorization flow actions', () => {
+  describe('when handling AUTHORIZATION_FLOW_REQUEST action', () => {
+    it('should add action to loading array', () => {
+      const requestAction = authorizationFlowRequest(
+        {} as Authorization,
+        AuthorizationAction.GRANT
+      )
+      expect(authorizationReducer(INITIAL_STATE, requestAction)).toEqual(
+        expect.objectContaining({
+          loading: [requestAction]
+        })
+      )
+    })
+
+    it('should remove error', () => {
+      const initialStateWithError = {
+        ...INITIAL_STATE,
+        authorizationFlowError: 'Some error'
+      }
+
+      expect(
+        authorizationReducer(
+          initialStateWithError,
+          authorizationFlowRequest(
+            {} as Authorization,
+            AuthorizationAction.GRANT
+          )
+        )
+      ).toEqual(
+        expect.objectContaining({
+          authorizationFlowError: null
+        })
+      )
+    })
+  })
+
+  describe('when handling AUTHORIZATION_FLOW_SUCCESS action', () => {
+    it('should remove action from loading array', () => {
+      const successAction = authorizationFlowSuccess({} as Authorization)
+      const initialStateWithLoading = {
+        ...INITIAL_STATE,
+        loading: [successAction]
+      }
+      expect(
+        authorizationReducer(initialStateWithLoading, successAction)
+      ).toEqual(
+        expect.objectContaining({
+          loading: []
+        })
+      )
+    })
+
+    it('should remove error', () => {
+      const initialStateWithError = {
+        ...INITIAL_STATE,
+        authorizationFlowError: 'Some error'
+      }
```

Comment:
What if we make the success action and the initial state global to the scope of the describe? By doing this, we could avoid repeating the initialization of the variable and initializing both resources in the describe.

```suggestion
    let successAction: AuthorizationSuccessAction
    let initialState: RootState
    beforeEach(() => {
      initialState = {
        ...INITIAL_STATE,
        loading: [successAction],
        authorizationFlowError: 'Some error'
      }
    })
    it('should remove action from loading array', () => {
      expect(
        authorizationReducer(initialStateWithLoading, successAction)
      ).toEqual(
        expect.objectContaining({
          loading: []
        })
      )
    })

    it('should remove error', () => {
```